applications/tasks/src/tasks_listener.erl

```diff
@@ -51,20 +61,83 @@ start_link() ->
                            ,[]
                            ).
 
+
+%%%===================================================================
+%%% AMQP API
+%%%===================================================================
+
 %%--------------------------------------------------------------------
-%% @doc Declare jobs available
+%% @public
+%% @doc
+%% @end
 %%--------------------------------------------------------------------
--spec help(kz_json:object(), kz_proplist()) -> 'ok'.
-help(JObj, _Props) ->
-    'true' = kapi_tasks:help_req_v(JObj),
-    RespJObj =
-        kz_json:from_list([{<<"Tasks">>, kz_json:from_list(knm_tasks:help())}
-                          ,{<<"Tasks-Category">>, knm_tasks:category()}
-                          ,{<<"Tasks-Module">>, knm_tasks:module()}
-                          ,{<<"Msg-ID">>, kz_api:msg_id(JObj)}
-                           | kz_api:default_headers(?APP_NAME, ?APP_VERSION)
-                          ]),
-    kapi_tasks:publish_help_resp(kz_api:server_id(JObj), RespJObj).
+-spec handle_lookup_req(kz_json:object(), kz_proplist()) -> 'ok'.
+handle_lookup_req(JObj, _Props) ->
+    'true' = kapi_tasks:lookup_req_v(JObj),
+    Help =
+        case
+            case {kapi_tasks:category(JObj), kapi_tasks:action(JObj)} of
```

Comment:
Addressed in https://github.com/2600hz/kazoo/pull/2409 along with `flushing tasks.#`